### ISSUE TITLE
Syntax error with Python 3.7

### ISSUE THREAD

**Issue by sekiyama58** [paulfitz/daff#115]
daff causes SyntaxError on launch with Python 3.7.

```
$ python3 --version
Python 3.7.0
$ pip3 install daff
$ daff
Traceback (most recent call last):
  File "/usr/local/bin/daff", line 7, in <module>
    from daff import main
  File "/usr/local/bin/daff.py", line 2723
    if io.async():
              ^
SyntaxError: invalid syntax
```

**Comment by sekiyama58** [comment]
In Python 3.7, `async` is reserved as a keyword, so cannot be used as function name.
It causes a SyntaxError exception.

**Comment by paulfitz** [comment]
Thanks for reporting this, working on a fix in #116.

**Comment by paulfitz** [comment]
Please reopen if problem persists in version 1.3.33 - cheers!
